packages/ens-controller/src/EnsController.ts

```diff
@@ -86,6 +118,25 @@ export class EnsController extends BaseControllerV2<
         ...state,
       },
     });
+
+    if (provider && onNetworkStateChange) {
+      onNetworkStateChange((networkState) => {
+        const currentNetwork =
+          networkState.network === 'loading' ? null : networkState.network;
+        if (currentNetwork && this.#getNetworkEnsSupport(currentNetwork)) {
+          const networkish: Networkish = {
```

Comment:
Nit: This type doesn't appear to be necessary

```suggestion
          const networkish = {
```